### ISSUE TITLE
Rendering ignoring line breaks

### ISSUE THREAD

**Issue by xmatthias** [microsoft/vscode-pull-request-github#3881]
- Extension version: 0.49.2022082209 
- VSCode Version: 1.70.2
- OS: linux

Steps to Reproduce:
Writing a comment with regular linebreaks (no double space at the end) will render wrong in vscode.

![image](https://user-images.githubusercontent.com/5024695/186074525-3c9f47d3-0f12-4a0c-bba0-cef760c2bf28.png)

in github - the linebreak is rendered properly.

![image](https://user-images.githubusercontent.com/5024695/186074537-20df22b2-89d6-4d72-8ad7-09898ed1c57f.png)

The actual content looks like this:

![image](https://user-images.githubusercontent.com/5024695/186074757-b4ac5ea4-4bf6-45d3-b9bf-eb52219f2933.png)

While the content is not proper markdown (which would require either an empty line or a double-space to render a linebreak) - i think this extension should align it's rendering to how github is rendering content.

**Comment by alexr00** [comment]
The rendering of the markdown in the comment peek view is done by VS Code, not the extension, so we can't change how it's rendered from the extension.

What we can do is convert single new lines to double new lines before passing them to VS Code to render. Happy to try that out at the beginning of the next iteration to see if folks using the pre-release build of the extension notice anything different.

**Comment by alexr00** [comment]
To verify:
- Leave a comment with single new lines and double new lines in a PR.
- Verify that the comment renders with the same new lines as on github.com
- Edit the comment, but don't actually change anything. Verify that the editable comment looks the same as the same editable comment on github.com.
- Verify that when you save you edited comment it renders the same as it did before editing.

**Comment by xmatthias** [comment]
For me this seems to work fine with v0.51.2022090509.

**Comment by alexr00** [comment]
It doesn't work when there's a space before or after the new lines.

**Comment by alexr00** [comment]
The "fix" for this also does bad things to code blocks. Reverting this change, and marking as upstream since this is how VS Code renders markdown.

**Comment by xmatthias** [comment]
@alexr00 how did we go from "this was implemented but causes a problem" to "not planned"?

i understand from the label it's upstream - but that'd at least imply a linked issue from upstream - no?

**Comment by alexr00** [comment]
@xmatthias I don't think we make any change upstream in VS Code for this either given that the current behavior is more correct.

The change that I implemented here used a heuristic, and it was ultimately a bad heuristic.

The change that would be required in VS Code would make markdown rendering less correct, which isn't a change I would support making. I've just opened https://github.com/microsoft/vscode/issues/165727 in case it would be acceptable.
